src/formatDate.js

```diff
@@ -50,7 +50,53 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
+
+  const resultArr = [];
+
+  const newObj = {
```

Comment:
```suggestion
  const newObject = {
```

don't cut names of variables